experiments/landlords/parent_company_search/How to Find Building Owners.ipynb

```diff
@@ -0,0 +1,675 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Finding Building Owners\n",
+    "\n",
+    "The problem: the City of Seattle does not provide us with the owners of buildings in the carbon benchmarking dataset. Without this data, we can't determine which companies have the largest environmental impact on Seattle. Moreover, the companies listed as building owners in the King County property database are often shell companies. There is no database where you can look up the parent company for a shell company. There is also no reliable way to find a parent company's shell companies (though this assumes you're looking for buildings owned by one particular company).\n",
+    "\n",
+    "**Vocabulary used in this tutorial**\n",
+    "\n",
+    "- Corporations and Charities Filings System: a publicly-available government database listing all companies and non-profits in Washington State.\n",
+    "- eRealProperty: a publicly-available government database that lists the owners of all real estate parcels in King County (where Seattle is located).\n",
+    "- Parent company: a company that owns subsidiary companies.\n",
+    "- Principal: a person who is an officer or director of a company. A company must list at least one principal to be incorporated.\n",
+    "- Shell company: a company owned by a parent company that disguises its relationship with that parent company. Creating a different shell company for every building owned by a parent company is a common tactic to hide the extent of a company's real estate holdings.\n",
+    "- Tax parcel owner: the result of looking up an address in eRealProperty. Also referred to as the Landlord of a building in this tutorial. \n",
```

Comment:
I think we should dump the landlord language. I realize I started it, but looking at it now, it doesn't make sense. We don't care about tenant-landlord relations in this, we're just focusing on owners. So I think we should do something like parcel_owner. What do you think?